Assets/AppCenter/Plugins/iOS/Auth/AuthUnity.mm

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#import "AuthUnity.h"
+#import <Foundation/Foundation.h>
+
+void* appcenter_unity_auth_get_type()
+{
+  return (void *)CFBridgingRetain([MSAuth class]);
+}
+
+void appcenter_unity_auth_sign_in_with_completion_handler(void(* handler)(MSUserInformation *, NSError *))
+{
+    [MSAuth signInWithCompletionHandler:^void(MSUserInformation *_Nullable userInformation, NSError *_Nullable error){
```

Comment:
Formatting

Reply:
I saw in other mm files (like AppCenterUnity.mm, CrashesUnity.mm) with the same format (don't have a new line for `{ ` ) of using native iOS function and return calling the handler, so I'm wondering should I keep it consistent with other similar calls? Thanks!